ivy/functional/ivy/experimental/elementwise.py

```diff
@@ -1380,6 +1380,8 @@ def sparsify_tensor(
     """
     if card >= ivy.prod(ivy.array(x.shape)):
         return x
+    elif card == 0:
+        return ivy.zeros_like(x)
```

Comment:
I don't think you need to explicitly handle the case when `card==0`, it will be handled by the code below :)

Reply:
I thought it would cause a bug because you can't concatenate a value with a none value

Reply:
I don't follow this completely. In the following line:

`ivy.concat([ivy.zeros(len(x) - card, dtype=x.dtype), x[-card:]], axis=0)` will be
`ivy.concat([ivy.zeros(len(x), dtype=x.dtype), x[0:]], axis=0)`